File: Servidor/InHomeTFGCarlosHernandez/src/main/java/org/tfg/inhometfgcarloshernandez/spring/model/EventoDTO.java
package org.tfg.inhometfgcarloshernandez.spring.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EventoDTO {
    private String tipo;
    private String nombre;
    private String votacion;
    private String horaComienzo;
    private String horaFin;
    private String organizador;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/response/EventoDTO.kt
package com.example.inhometfgandroidcarloshernandez.data.model.response

data class EventoDTO (
    val id : Int,
    val tipo: String,
    val nombre: String,
    val votacion: String,
    val horaComienzo: String,
    val horaFin: String,
    val organizador: String,
)